rs/ic_os/config/src/config_ini.rs

```diff
@@ -94,7 +101,12 @@ pub fn get_config_ini_settings(config_file_path: &Path) -> Result<ConfigIniSetti
 
     let domain = config_map.get("domain").cloned();
 
-    let network_settings = NetworkSettings {
+    let verbose = config_map
+        .get("verbose")
+        .map(|s| s.eq_ignore_ascii_case("true"))
```

Comment:
super nit: is_some_and

Reply:
Much nicer! Fixed in https://github.com/dfinity/ic/pull/1539